admin/includes/init_includes/init_admin_auth.php

```diff
@@ -29,6 +29,7 @@
 // admin folder rename required
 if (!defined('ADMIN_BLOCK_WARNING_OVERRIDE') || ADMIN_BLOCK_WARNING_OVERRIDE == '')
 {
+  define('ADMIN_BLOCK_WARNING_OVERRIDE', false);

```

Comment:
Hmmm ... if we get here via the `!defined()` part, this is okay. But if we get here by the `== ''` part then this will throw an error about already being defined, won't it?

Reply:
that does sound correct.